### ISSUE TITLE
Delete land parcels on completion of map pack generation

### ISSUE THREAD

**Issue by CrispinF** [exeGesIS-SDM/ELM#38]
Discussed by email, and to be prioritised by @sarahpoppy 
I noted "This would make further clean-up jobs easier, as only the map PDFs would need to be deleted, plus it reduces the size bloat of the database as more applications are processed."

**Comment by CrispinF** [comment]
This seems to be becoming a high priority requirement... for @sarahpoppy to confirm?

**Comment by sarahpoppy** [comment]
Yes, you asked me to prioritise, and given we've said that is what we'll do, we should

**Comment by epennifold** [comment]
Confirmed in meeting with SP - parcels to be deleted immediately on creation of map pack.
